### PR TITLE
vimiv-qt: init at 0.8.0

### PR DESCRIPTION
**Pull request by dschrempf**
###### Motivation for this change
Vimiv is an image viewer with Vim-like keybindings. A Qt version has long been developed and is meant to superseed the old and now outdated GTK version (which is already in Nixpkgs). This pull request adds the newer Qt based `vimiv-qt`.

Please review carefully! This is my first contribution to Nixpkgs.

In particular, I have the following questions:
- At the moment updates have to be done manually. I am not monitoring the development of `vimiv-qt`. How will I know when the package is outdated? I know that there are automated ways using bots. What is the preferrable way to get notified about updates as a maintainer?
- I manually added the commands of the [make file](https://github.com/karlch/vimiv-qt/blob/master/misc/Makefile) into `postInstall`. Please let me know if this is OK! 

Thank you!

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
> At the moment updates have to be done manually. I am not monitoring the development of vimiv-qt. How will I know when the package is outdated? I know that there are automated ways using bots. What is the preferrable way to get notified about updates as a maintainer?

There is @r-ryantm and you can monitor https://repology.org/. There are no notifications for updates.

File: maintainers/maintainer-list.nix
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:10777]: Please put this into a separate commit with the message ``maintainers: add dschrempf``.

```suggestion
  dschrempf = {
```

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/graphics/vimiv-qt/default.nix:38]:
```suggestion
      wrapQtApp $out/bin/vimiv
```

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/graphics/vimiv-qt/default.nix:35]:
```suggestion
  # Vimiv has to be wrapped manually because it is a non-ELF executable.
```

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/graphics/vimiv-qt/default.nix:31]:
```suggestion
    install -Dm644 misc/vimiv.desktop $out/applications/vimiv.desktop
    install -Dm644 misc/org.karlch.vimiv.qt.metainfo.xml $out/metainfo/org.karlch.vimiv.qt.metainfo.xml
    install -Dm644 LICENSE $out/licenses/vimiv/LICENSE
    install -Dm644 icons/vimiv.svg $out/icons/hicolor/scalable/apps/vimiv.svg
    installManPage misc/vimiv.1

    for i in 16 32 64 128 256 512; do
      install -Dm644 icons/vimiv_''${i}x''${i}.png $out/icons/hicolor/''${i}x''${i}/apps/vimiv.png
    done
```

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/graphics/vimiv-qt/default.nix:21]:
```suggestion
  nativeBuildInputs = with python39Packages; [ wrapQtAppsHook setuptools ];

  propagatedBuildInputs = with python39Packages; [ pyqt5 py3exiv2 qtsvg ];
```

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/graphics/vimiv-qt/default.nix:42]: Please add a description.

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/graphics/vimiv-qt/default.nix:8]:
```suggestion
python39Packages.buildPythonApplication rec {
```

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/graphics/vimiv-qt/default.nix:5]:
```suggestion
{ stdenv
, lib
, fetchFromGitHub
, python39Packages
, qt5
}:
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please squash the package and maintainer related commits together.

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (dschrempf) [reply pkgs/applications/graphics/vimiv-qt/default.nix]: Done.

**Comment by dschrempf** [comment]
Thank you very much for the review! I hope that I could address all of the suggested changes!

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (dschrempf) [reply pkgs/applications/graphics/vimiv-qt/default.nix]: Actually, this change breaks the package. `wrapQtAppsHook` and `qtsvg` are part of `qt5`. Should I revert your suggested removal of the `qt5` prefixes?

Comment (SuperSandro2000) [reply pkgs/applications/graphics/vimiv-qt/default.nix]: Updated the suggestion to not remove the prefix.

**Comment by dschrempf** [comment]
BTW, is there a reason why the quotation marks have been removed? If $out contains, let's say, spaces, the script would break. But I guess, this will never be the case.

**Comment by SuperSandro2000** [comment]
> If $out contains, let's say, spaces, the script would break. But I guess, this will never be the case.

$out cannot contain spaces because the nix store up to that path cannot contain spaces.

File: maintainers/maintainer-list.nix
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:10777]: Please sort your entry alphabetically.

Comment (dschrempf) [reply maintainers/maintainer-list.nix]: Done! I guess the confusion arose because the last 10 entries or so are not sorted, and I assumed I had to add myself at the end.

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/graphics/vimiv-qt/default.nix:5]:
```suggestion
, qt5
, installShellFiles
```

required for installManPage

**Review by SuperSandro2000 — APPROVED** [review]
other than that LGTM if the build succeeds.

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (dschrempf) [reply pkgs/applications/graphics/vimiv-qt/default.nix:5]: Done with the last commit, sorry forgot to resolve the conversation.

**Comment by dschrempf** [comment]
Hi,

just wanted to ask if there is anything else I can do? Otherwise, please merge or close the pull request! Thanks!

**Comment by wedens** [comment]
I've tried using it and haven't had any issues so far.

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (wedens) [inline pkgs/applications/graphics/vimiv-qt/default.nix:25]:
```suggestion
    install -Dm644 misc/vimiv.desktop $out/share/applications/vimiv.desktop
```

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/graphics/vimiv-qt/default.nix:4]: Please don't hardcode the python version unless it is absolutely necessary. If the package works on 3.8 python3 should be used.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (dschrempf) [reply pkgs/applications/graphics/vimiv-qt/default.nix]: `python3.pkgs.py3exiv2` is broken on 20.09, but I guess this does not matter here. It works on `master`, and `unstable`.

I hope it works :)!

**Comment by dschrempf** [comment]
Somehow it still says that changes are requested, but I cannot find any more requests. Let me know, if I missed something!

File: pkgs/applications/graphics/vimiv-qt/default.nix
Comment (SuperSandro2000) [inline pkgs/applications/graphics/vimiv-qt/default.nix:20]:
```suggestion
  nativeBuildInputs = [ installShellFiles qt5.wrapQtAppsHook python3.pkgs.setuptools ];
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request merged**
